hw/drivers/display/cfb/include/display/cfb.h

```diff
@@ -42,17 +42,17 @@ enum cfb_display_param {
 };
 
 enum cfb_font_caps {
-	CFB_FONT_MONO_VPACKED		= BIT(0),
-	CFB_FONT_MONO_HPACKED		= BIT(1),
+	CFB_FONT_MONO_VPACKED		= (1UL << (0)),
```

Comment:
I guess this is macro code extracted. Parentheses make sense in macros bur here 0 could be just 0 instead of (0). Other parts of zephyr ported code do not uses extra () for constants.